ui/pages/token-allowance/index.scss

```diff
@@ -38,4 +38,29 @@
   &__full-tx-content {
     max-width: 100%;
   }
+
+  &__custom-nonce-warning {
+    width: 100%;
+    height: 30px;
+  }
+
+  &__custom-nonce-content {
+    height: 49px;
+    border: 1px solid var(--color-border-muted);
+    box-sizing: border-box;
+  }
+
+  &__custom-nonce-header {
+    flex: 1;
+  }
+
+  &__custom-nonce-value {
+    flex: 0;
+  }
+
+  &__custom-nonce-edit {
+    @include H7;
+
+    width: auto;
+  }
```

Comment:
`@include H7` should not be needed in new `Text` component.

Reply:
@jpuri I moved this styles into `CustomNonce` component. This is important because this `@incluce H7` is for `<Button/>` size (`Edit` button). You can check changes [here](https://github.com/MetaMask/metamask-extension/pull/17945/commits/cb62dbcddb0b653bb306db0c89f7e9bc72cd4284).